save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -175,10 +177,19 @@ class DockerService(private val configProperties: ConfigProperties) {
         val testSuitesDir = resourcesPath.resolve(standardTestSuiteDir)
         // copy corresponding standard test suites to resourcesRootPath dir
         copyTestSuitesToResourcesPath(testSuitesForDocker, testSuitesDir)
+
+        // move additional files, which were downloaded into the root dit to the execution dir for standard suites
+        execution.additionalFiles?.split(";")?.filter { it.isNotBlank() }?.forEach {
+            val additionalFilePath = Paths.get(resourcesPath.resolve(File(it).name).absolutePath)
```

Comment:
Do we always have additional files in the root of resourcesPath? Maybe, having `resolve(it)` will be more reliable?

Reply:
Actually, in `execution.additionalFiles` we have the full paths to the files from our storage `/home/cnb/files/...`, but we also know, that during `uploadBin` execution we have already this files in the `resourcesPath`, with the same `.name` as in `execution.additionalFiles`

Reply:
OK